ROSCO_toolbox/utilities.py

```diff
@@ -209,6 +210,13 @@ def write_DISCON(turbine, controller, param_file='DISCON.IN', txt_filename='Cp_C
     file.write('!------- Pitch Actuator Faults -----------------------------------------------------\n')
     file.write('{}                ! PF_Offsets     - Constant blade pitch offsets for blades 1-3 [rad]\n'.format(''.join('{:<10.8f} '.format(rosco_vt['PF_Offsets'][i]) for i in range(3))))
     file.write('\n')
+    file.write('!------- Active Wake Control -----------------------------------------------------\n')
+    file.write('{0:<12d}          ! AWC_NumModes       - Number of AWC modes\n'.format(int(rosco_vt['AWC_NumModes'])))
+    file.write('{}                ! AWC_n              - AWC azimuthal number, must be an array the size of AWC_NumModes\n'.format(write_array(rosco_vt['AWC_n'],'<4d')))
+    file.write('{}                ! AWC_omega          - AWC frequency, must be an array the size of AWC_NumModes [rad/s]\n'.format(write_array(rosco_vt['AWC_omega'],'<6.4f')))
+    file.write('{}                ! AWC_amp            - AWC amplitude, must be an array the size of AWC_NumModes [rad]\n'.format(write_array(rosco_vt['AWC_amp'],'<6.4f')))
+    file.write('{}                ! AWC_clockangle     - AWC clock angle, must be an array the size of AWC_NumModes [deg]\n'.format(write_array(rosco_vt['AWC_clockangle'],'<6.4f')))
+    file.write('\n')
```

Comment:
Can you expand on these input descriptions so it's clear to the user what they do?  I'm not sure what azimuthal number means, e.g...

Reply:
@kbrown1snl, this is the best place to update input file descriptions.  Your readme might have helpful info for users to place here.

Reply:
@dzalkind, I transferred some of the language to utilities.py. Hopefully there's not too much text there now.